controllers/machineset_delete_policy.go

```diff
@@ -33,11 +33,8 @@ type (
 const (
 	// DeleteNodeAnnotation marks nodes that will be given priority for deletion
 	// when a machineset scales down. This annotation is given top priority on all delete policies.
-	// Deprecated: Please use DeleteMachineAnnotation instead.
+	// Deprecated: Please use DeleteMachineAnnotation instead under api/v1alpha4 package.
```

Comment:
Revert this, after #3955 merges, we should rebase and this deprecated annotation is going to be removed from the codebase